templates/microApp/src/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <div id="app"></div>
+  <script>
+    window.onload = () => {
+      const LIB_NAME = 'remoteApp'
```

Comment:
Can we not do a constant replacement here, based on whats the app name given to generator?